cookbook/test/integration/environments/stage.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "stage",
+  "json_class": "Chef::Environment",
+  "chef_type": "environment"
+}
```

Comment:
Ehhhh...while that's true, not everything accepts _default as an environment and stage doesn't really hurt anything per se...